application/app/config/applications/mapbender_mobile.yml

```diff
@@ -151,13 +151,13 @@ parameters:
                     imprint:
                         title: Imprint
                         class: Mapbender\CoreBundle\Element\HTMLElement
-                        content: '<div style="color: #6FB536; font-weight:bold" >powered by Mapbender. Read more about Mapbender3 at the <a href="http://mapbender3.org" target="_blank">Mapbender3 Project Page</a>.</div>'
+                        content: '<div style="color: #6FB536; font-weight:bold" >powered by Mapbender. Read more about Mapbender at the <a href="https://www.mapbender.org" target="_blank">Mapbender Project Page</a>.</div>'
                         classes: html-element-inline
 
                     help:
                         title: Help
                         class: Mapbender\CoreBundle\Element\HTMLElement
-                        content: 'Enjoy the new Mapbender mobile Template. Please note that not all elements can be used in the mobile application. <div style="color: #6FB536; font-weight:bold">Find out more about the Mapbender mobile in the <a href="http://doc.mapbender3.org/en/book/mapbender_templates.html" target="_blank">Template Documentation</a>.</div>'
+                        content: 'Enjoy the new Mapbender mobile Template. Please note that not all elements can be used in the mobile application. <div style="color: #6FB536; font-weight:bold">Find out more about the Mapbender mobile in the <a href="https://doc.mapbender.org/en/book/mapbender_templates.html" target="_blank">Template Documentation</a>.</div>'
```

Comment:
We have to change the link, after the restructuring of the MB documentation. 
Ticket: https://github.com/mapbender/mapbender/issues/725
Otherwise no complaints.